site/.vitepress/config.ts

```diff
@@ -610,11 +596,8 @@ export default defineConfig({
             text: "AccountSigner",
             collapsed: true,
             base: "/packages/aa-ethers/account-signer",
+            link: "/introduction",
             items: [
-              {
-                text: "Overview of AccountSigner",
-                link: "/introduction",
-              },
```

Comment:
let's put this back, per my comment re: no section header pages to make each page explicitly visible on sidebar